Source/Public/Uninstall-BicepCLI.ps1

```diff
@@ -0,0 +1,48 @@
+function Uninstall-BicepCLI {
+    [CmdLetBinding()]
+    param (
+        [switch]$Force
+    )
+    
+    # Test if we are running as administrators.
+    $currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())
+    $IsAdmin = $currentPrincipal.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+
+    if (-not $IsAdmin -and -not $Force) {
+        Write-Error 'Some Bicep parts might not be properly uninstalled unless you run elevated. Use the -Force switch to try anyway.'
+    }
+    if (-not $IsAdmin -and $Force) {
+        Write-Host 'You are not running elevated. We may not be able to remove all parts.'
+    }
+    if ($IsAdmin -or $Force) {    
+        $UninstallerFileName = 'unins000.exe'
+        $BicepExeName = 'bicep.exe'
+        $BicepInstalls = $env:Path -split ';' | Where-Object {$_ -like "*\.bicep" -or $_ -like "*\Bicep CLI"} 
+
+        foreach ($Install in $BicepInstalls) {
+            $FileContents = Get-ChildItem $Install
+            if (($UninstallerFileName -in $FileContents.Name) -and ($BicepExeName -in $FileContents.Name)) {
+                # Bicep is installed using installer. Try using it to uninstall
+                $UninstallerPath = ($FileContents | Where-Object -Property Name -eq $UninstallerFileName).FullName
+                & $UninstallerPath /VERYSILENT
+            }
+            else {
+                # Bicep is running in standalone exe mode. Remove manualy
+                $ExePath = Join-path -Path $Install -ChildPath $BicepExeName
+                if (Test-Path $ExePath) {
+                    Remove-Item $ExePath
+                }
+            }
+        }
+
```

Comment:
The test to verify if uninstall was successful or not runs to quick after the uninstaller is finished. The terminal session still has bicep loaded as a command for a couple of seconds. Got it to work as expected by adding `Start-Sleep 5` before the validation.

Reply:
fixed in commit https://github.com/StefanIvemo/BicepPowerShell/pull/14/commits/fb98e1b83fbee83f0696a29628722617e5e10edb